src/Device/Driver/OpenVario.cpp

```diff
@@ -131,13 +228,33 @@ OpenVarioDevice::POV(NMEAInputLine &line, NMEAInfo &info)
    * R: total pressure in hPa
    * S: true airspeed in km/h
    * T: temperature in deg C
+   * ?: respond with selected strings, e.g. Ballast, Bugs, Polar
+   *    Example: $POV,?,RPO,MC,WL*2E means: Real Polar, MacCready, Wing Load
    */
 
+
   while (!line.IsEmpty()) {
     char type = line.ReadOneChar();
     if (type == '\0')
       break;
 
+    if (type == '?') {
+      NullOperationEnvironment env;
+      char query_item[5];
+
+      for (int i=0;i < 10;i++) { // not more than 10 loops!
+	line.Read(query_item,sizeof(query_item));
+	if (strlen(query_item) == 0) return true;
+
+	if (StringIsEqual(query_item,"WL")) RepeatBallast(env);
+	if (StringIsEqual(query_item,"BU")) RepeatBugs(env);
+	if (StringIsEqual(query_item,"MC")) RepeatMacCready(env);
+	if (StringIsEqual(query_item,"IPO") && _ideal_polar_valid) ComposeWrite('I',_ideal_polar, env);
+	if (StringIsEqual(query_item,"RPO") && _real_polar_valid) ComposeWrite('R',_real_polar, env);
```

Comment:
Something that bothers me here is: if polar is not available on request, we just keep silent and not answering anything. This might be confusing and make debugging difficult. I suggest responding with NMEA sentence like `POV,C,NA,IPO` in this case (and other cases too). This will make it obvious (for a human) that request cannot be fullfulled at this moment and will not confuse existing openvario nmea parsers.

Reply:
That's a good point. But I'm not convinced n/a is a better choice. This might make the receiving end choke. BTW: the same happens for MC, BU, WL, it's hidden behind the Repeat... functions.

Anybody else an opinion?

Reply:
I think we can safely assume that receiver will ignore unknown/unrecognized sentences. We can pick different response format, but I believe we have to answer something.